simulationTools/sim_coherenceOrder.m

```diff
@@ -0,0 +1,31 @@
+%sim_coherenceOrder.m
+%Dana Goerzen and Jamie Near, 2021.
+%
+% USAGE:
+% out = sim_coherenceOrder(spinSys);
+%
+% DESCRIPTION:
+% Creates a n x n matrix corresponding to the coherence order of each
+% proton in spin system. For use in nulling signal incoherences during
```

Comment:
I would rephrase this first sentence to say that it "Creates an n x n matrix that represents the coherence order of each element of the density matrix for the given spin system."

Reply:
Done